frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -317,10 +318,10 @@ const TableContainer = <T,>({
                   {actionButton.iconSvg && <Icon name={actionButton.iconSvg} />}
                 </>
               </Button>
-            )}
-            {customControl && customControl()}
-          </div>
-          <div className="box search">
+            </div>
+          )}
+          <div className="stackable-header top-shift-header">
+            {customControl ? customControl() : undefined}
```

Comment:
Guessing there's a good reason I should be aware of for the future for why we aren't using the `&&` syntax here?

Reply:
nope! >.<